contracts/interfaces/IMasterAwareV2.sol

```diff
@@ -13,7 +13,8 @@ interface IMasterAwareV2 {
     AS, // Assessment.sol
     TK, // NXMToken.sol
     PS, // LegacyPooledStaking.sol
-    GV  // Governance.sol
+    GV,  // Governance.sol
+    IC  // IndividualClaims.sol
```

Comment:
I think we may have a conflict here. `IC` is used in production for the `Incidents`. IIRC Dragos intended to "take over" the abbreviation and reuse it for the other contract but I'm really unsure here.